descriptor.md

```diff
@@ -0,0 +1,117 @@
+# OpenContainers Content Descriptors
+
+OCI have several components that come to together to describe an image.
+References between components form a [Merkle Directed Acyclic Graph (DAG)](https://en.wikipedia.org/wiki/Merkle_tree).
+The references in the _Merkle DAG_ are expressed through _Content Descriptors_.
+A _Content Descriptor_ or _Descriptor_, describes the disposition of targeted content.
+A _Descriptor_ includes the type of content, an independently-verifiable content identifier, known as a "digest" and the byte-size of the raw content.
```

Comment:
The “, known as a "digest"” bit makes the three-part content listing hard for me to read.  Can we drop this sentence, since we have a detailed “Properties” section below?